examples/gno.land/p/demo/avl/rolist/rolist.gno

```diff
@@ -52,6 +53,7 @@ type IReadOnlyList interface {
 	Get(index int) any
 	Slice(startIndex, endIndex int) []any
 	ForEach(fn func(index int, value any) bool)
+	ReadOnlyTree() *rotree.ReadOnlyTree
```

Comment:
```suggestion
```

IReadonlyList should not require us to depend on a tree. The goal of this interface is to enable alternative implementations.

Reply:
Make sense, you're right 👍 
Done https://github.com/gnolang/gno/pull/4537/commits/2df66c11e584940f71337716be150d501b865294